userspace/libsinsp/plugin.cpp

```diff
@@ -712,6 +713,28 @@ void sinsp_plugin::validate_init_config_json_schema(std::string& config, std::st
 	}
 }
 
+bool sinsp_plugin::set_config(const std::string& config)
+{
+	if(!m_inited)
+	{
+		throw sinsp_exception(std::string(s_not_init_err) + ": " + m_name);
+	}
+
+	std::string conf = config;
+	validate_config(conf);
+
+	ss_plugin_rc rc = SS_PLUGIN_FAILURE;
+	ss_plugin_set_config_input input;
+	input.config = conf.c_str();
+
+	if(m_handle->api.set_config)
+	{
+		rc = m_handle->api.set_config(m_state, &input);
+	}
+
+	return rc == SS_PLUGIN_SUCCESS;
```

Comment:
```suggestion
	ss_plugin_set_config_input input;
	input.config = conf.c_str();

	if(!m_handle->api.set_config)
	{
		return false;
	}

	return m_handle->api.set_config(m_state, &input) == SS_PLUGIN_SUCCESS;
```

If there is no API symbol, I guess we should let users know that the config has not been set.